databricks/README.md

```diff
@@ -0,0 +1,70 @@
+# Agent Check: Databricks
+
+## Overview
+
+This check monitors [Databricks][1].
+
+## Setup
+
+### Installation
+
+The Databricks check is included in the [Datadog Agent][2] package.
```

Comment:
Is this the correct link? Seems to point to a blog article rather than [datadog agent link](https://docs.datadoghq.com/agent/).

Reply:
Yes the intention is to reference the recommended/most publicized way we currently suggest monitoring Databricks